tools/pipelines/repo-policy-check.yml

```diff
@@ -30,19 +30,19 @@ resources:
     ref: refs/tags/release
 extends:
   # The pipeline extends the 1ES PT which will inject different SDL and compliance tasks.
```

Comment:
Please update this comment with more details about why public pipelines are different.

Reply:
Done